universal-login-contracts/contracts/ENSRegistered.sol

```diff
@@ -9,7 +9,7 @@ import "@ensdomains/resolver/contracts/PublicResolver.sol";
 contract ENSRegistered {
     bytes32 constant ADDR_REVERSE_NODE = 0x91d1777781884d03a6757a803996e38de2a42967fb37eeaca72729271025a9e2;
 
-    function registerENS(bytes32 _hashLabel, string memory _name, bytes32 _node, ENS ens, FIFSRegistrar registrar, PublicResolver resolver) internal {
+    function registerENS(bytes32 _hashLabel, string memory _name, bytes32 _node, ENS ens, FIFSRegistrar registrar, PublicResolver resolver) public {
```

Comment:
This can have serious security implications, as registrar and resolver use `msg.sender`. Which means now anyone can register domains in the name of the contract instance. I propose:
- adding a `constructor` instead of `registerENS` or
- add `ENSRegisterUnderTest` with a public method